src/components/data/DataTable.js

```diff
@@ -233,6 +235,7 @@ const DataTable = props => {
                   }
                 },
                 {
+                  field: 'name',
```

Comment:
Are the single quotes around `name` here intentional? If so, what's the difference between this field and the other fields where name isn't enclosed in single quotes?

Reply:
No difference. I agree they might as well be consistent. Is there a preferred quote type (single vs double) in the repo?

Reply:
Ah. Here's the name of the field is `'name'` to match up with the `Sortable` definition attached to the column. In the situation below, `name` is a variable

Reply:
I agree this is confusing, but I just pulled up the column names that were already in the corresponding `Sortable` definitions.

Reply:
Ah okay I think it's fine then, just wanted to double check that this wasn't meant to be a variable like it was down below

Reply:
Yeah, it's a good question. Thanks for double checking